dowhy/causal_estimators/distance_matching_estimator.py

```diff
@@ -67,22 +151,16 @@ def __init__(self, *args, num_matches_per_unit=1, distance_metric="minkowski", e
             self.logger.error(error_msg)
             raise Exception(error_msg)
 
-        # Dictionary of any user-provided params for the distance metric
-        # that will be passed to sklearn nearestneighbors
-        self.distance_metric_params = {}
-        for param_name in self.Valid_Dist_Metric_Params:
-            param_val = getattr(self, param_name, None)
-            if param_val is not None:
-                self.distance_metric_params[param_name] = param_val
-
-        self.logger.info("INFO: Using Distance Matching Estimator")
         self.symbolic_estimator = self.construct_symbolic_estimator(self._target_estimand)
         self.logger.info(self.symbolic_estimator)
-        self.matched_indices_att = None
-        self.matched_indices_atc = None
 
-    def _estimate_effect(self):
+        return self
+
+    def estimate_effect(self, treatment_value: Any = 1, control_value: Any = 0, target_units=None, **_):
```

Comment:
I like it, I just didn't know if we actually wanted to rename it. I'll rename it in the estimators but leave the estimate_effect method name in the CausalModel otherwise we might break backwards compatibility.

Reply:
That sounds good!

Reply:
Will rename in future PR as there are other places where the `effect()` function exist.